public/resources/en/sessions.json

```
@@ -2,10 +2,10 @@
   "title": "Sessions",
   "sessionExplain": "Sessions are sessions of users in this realm and the clients that they access within the session.",
   "searchForSession": "Search session",
-  "subject": "Subject",
-  "lastAccess": "Last access",
-  "startDate": "Start date",
-  "accessedClients": "Accessed clients",
+  "user": "User",
+  "lastAccess": "Last Access",
```

everywhere else we use lower case so this would need to be:
```suggestion
  "lastAccess": "Last access",
```